docs/cmdline-opts/ipfs-gateway.d

```diff
@@ -0,0 +1,13 @@
+c: Copyright (C) 2022, Mark Gaiser, <mark.gaiser@protocol.ai>
+SPDX-License-Identifier: curl
+Long: ipfs-gateway
+Arg: <URL>
+Help: Defines the gateway to handle IPFS protocol requests.
+Added: 7.86.0
+See-also: help manual
+Category: ipfs
+Example: --ipfs-gateway http://localhost:8080 $URL
+---
+Specifies which gateway to use for resolving ipfs:// and ipns://.
+Not specifying this argument will let cURL try to automatically
+find a local gateway in your homefolder in ~/.ipfs/gateway.
```

Comment:
The gateway should be specified as a URL, preferably with a `https://` prefix, right? Maybe that should be made clearer?

Not sure if it needs to, but since that then makes a HTTPS request under the hood curl is subject to the regular certificate checks and handling, treating the gateway as a server, which also might help users to understand how to work with this.

Reply:
We do prefer a local node and that is http.

I've updated this locally to include an example for local gateway usage, an ipfs.io example and a reference to a link of public gateways. You'll see it in the next update.